view/frontend/templates/certificates/list.phtml

```diff
@@ -0,0 +1,70 @@
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @copyright  Copyright (c) 2018 Avalara, Inc.
+ * @license    http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+/** @var \ClassyLlama\AvaTax\Block\Adminhtml\CustomerCertificates $block */
+?>
+<div class="table-wrapper">
+    <table class="data table">
+        <thead>
+            <tr>
+                <th class="data-grid-th"><?php echo __('State') ?></th>
+                <th class="data-grid-th"><?php echo __('Signed Date') ?></th>
+                <th class="data-grid-th"><?php echo __('Expiration Date') ?></th>
+                <th class="data-grid-th"><?php echo __('Status') ?></th>
+                <th class="data-grid-th" colspan="3">&nbsp;</th>
+            </tr>
+        </thead>
+
+        <tbody>
+            <?php foreach ($block->getCertificates() as $certificate): ?>
+                <tr data-role="row">
+                    <td data-th="<?php echo __('State') ?>"><?php echo $certificate->getData('exposure_zone')->getData(
+                            'name'
+                        ); ?></td>
+                    <td data-th="<?php echo __('Signed Date') ?>"><?php echo $certificate->getData(
+                            'signed_date'
+                        ); ?></td>
+                    <td data-th="<?php echo __('Expiration Date') ?>"><?php echo $certificate->getData(
+                            'expiration_date'
+                        ); ?></td>
+                    <td data-th="<?php echo __('Status') ?>"><?php echo $certificate->getData('valid') ? 'Approved'
+                            : 'Denied'; ?></td>
+                    <td>
+                        <?php if ($certificate->getData('valid')): ?>
+                            <a target="_blank"
+                               href="<?php echo $block->getCertificateUrl($certificate->getData('id')) ?>">View
+                                Certificate</a>
+                        <?php endif ?>
+                    </td>
+                    <td>
+                        <?php if ($certificate->getData('valid')): ?>
+                            <a href="javascript:void(0)">Renew Certificate</a>
+                        <?php endif ?>
+                    </td>
+                    <td>
+                        <?php if ($certificate->getData('valid')): ?>
+                            <a href="javascript:void(0)">Delete Certificate</a>
+                        <?php endif ?>
+                    </td>
+                </tr>
+            <?php endforeach; ?>
+
+            <?php if (count($block->getCertificates()) === 0): ?>
+                <tr class="data-grid-tr-no-data even">
+                    <td class="empty-text" colspan="7">No certificates found for this customer</td>
```

Comment:
@LordZardeck - Another pieces of text that needs to be translated.  (I may have missed this one in the admin template.)